package.json

```diff
@@ -84,6 +84,7 @@
     "react-cookie": "^4.1.1",
     "react-dom": "^18.2.0",
     "react-stately": "^3.23.0",
+    "requests": "^0.3.0",
```

Comment:
Commented in https://github.com/mozilla/blurts-server/pull/3397#discussion_r1327645030, but should this be request singular or plural? It was the singular "request" dependency in the other PR.